### PR TITLE
Facehuggers no longer knock you out for 2 minutes straight and instead will paralyze you and cause you to scream.

### PR DESCRIPTION
**Pull request by Iamgoofball**
## About The Pull Request

Facehuggers no longer knock you out for 2 minutes straight and instead will paralyze you and cause you to scream, allowing counterplay to exist beyond just being a prophet and having future sight to know when a xeno is gonna pop out of a vent. I mean, damn, I get fired today and decide ok I'll play some Space Station 13 and what do I gotta deal with? The same trash unfixed broken antag mechanic that's been in the game for 15 years that you people won't let us remove because soul or something, this shit just ruined my entire night, fuck xenos, fuck this antag type, and fuck 2 minute hard stuns with zero counter that can be applied instantly by someone who just spent the last 5 minutes sitting in a vent camping to ruin someone's round.

Every time I play this fucking game xenomorphs ruin it for me. I swear to god. Every single time.

## Why It's Good For The Game

Facehuggers no longer knock you out for 2 minutes straight and instead will paralyze you and cause you to scream, allowing counterplay to exist beyond just being a prophet and having future sight to know when a xeno is gonna pop out of a vent. I mean, damn, I get fired today and decide ok I'll play some Space Station 13 and what do I gotta deal with? The same trash unfixed broken antag mechanic that's been in the game for 15 years that you people won't let us remove because soul or something, this shit just ruined my entire night, fuck xenos, fuck this antag type, and fuck 2 minute hard stuns with zero counter that can be applied instantly by someone who just spent the last 5 minutes sitting in a vent camping to ruin someone's round.

## Changelog

:cl:
balance: Facehuggers no longer knock you out for 2 minutes straight and instead will paralyze you and cause you to scream, allowing more counterplay.
/:cl:

cobby edit: CL

### CODE REVIEW

**Comment by Ghommie** [comment]
Biggest "i ded" PR I've ever seen. The title says all.

**Comment by Jacquerel** [comment]
> Biggest "i ded" PR I've ever seen. The title says all.
> 
> On a itsy more serious note, please refrain from making such absurdly long PR titles in the future. You can vent in the comment for it.

Might want to edit the changelog too, on the other hand it is kinda funny.

**Comment by Iamgoofball** [comment]
the changelog provides context for players questioning the change

**Comment by Rhials** [comment]
Shoulda wore a helmet!

**Comment by GuillaumePrata** [comment]
This punishes, but won't stop the annoying vent meta we have with Xenos, they can still throw a hugger on you and leave again, spending 5 seconds of their time to force the crew to run to medbay or be round removed, while nerfing hard on their nest defense.

We should just make that Xenos can't go on vents while carrying a hugger, they can still throw huggers at people assaulting the nest or the random assistant ~~valid hunting~~ exploring maints, but it would completely end the annoying hit and run tactics Xenos do with vents and huggers.

Swarmers were removed in part because of this, vent hit and run tactics drag the round to a halt and make people give up on beating the antags and just calling the shuttle instead, we should just remove these ~~xenomorphs~~ strats.

**Comment by ExcessiveUseOfCobblestone** [comment]
Also dont @ me but vents should act as "closed" when turned off, which makes some mobs have unable to pass through the vent or require them to pry it open which gives a delay and sound cue. There should be an intermediate between wide open and welded shut that isnt such a pain to fix after the threat is eliminated, but doesnt hardstop the threat from utilizing the unwelded vent.

**Review by MrMelbert — CHANGES_REQUESTED** [review]
I am not opposed to this but it could use a few tweaks to make it less dead easy to immediately slam `; XENOS MEDBAY MAINT` after being hugged

Such as
- You can't talk while facehugged, or all talking gets muffled (`; MMM MMMM MMM`)
- You can talk, but you can't use your radio
- Same as above but it forces you to use whisper 
- Any combination of above (forced whisper + muffled + no radio sounds funny to me) 

And
- You can't remove a facehugger from your face by yourself without help until it's dead

**Comment by MrMelbert** [comment]
(If ur lazy you can just give facehuggers `BLOCKS_SPEECH` like muzzles but that's kinda boring)

**Comment by IndieanaJones** [comment]
I would just make it so they can't ventcrawl while holding an item like monkeys.

**Comment by bigfatbananacyclops** [comment]
i dont typically advocate 'ided' prs. but xenos are in fact, and have been broken for 15 years. it wouldnt be bad to atleast be able to stop a hugger.

**Comment by Zenog400** [comment]
> i dont typically advocate 'ided' prs. but xenos are in fact, and have been broken for 15 years. it wouldnt be bad to atleast be able to stop a hugger.

You know that spacesuit helmets and bio suit helmets all stop them, right?

**Comment by bigfatbananacyclops** [comment]
> > i dont typically advocate 'ided' prs. but xenos are in fact, and have been broken for 15 years. it wouldnt be bad to atleast be able to stop a hugger.
> 
> 
> 
> You know that spacesuit helmets and bio suit helmets all stop them, right?

Yes i know and i know more than you. But this will atleast help relieve the immense power xenos have.

**Comment by ExcessiveUseOfCobblestone** [comment]
They're not intended to be a fair fight, theyre meant to be used to wrap up the round.

**Comment by Iamgoofball** [comment]
@ExcessiveUseOfCobblestone getting killed fast is fine, staring at a black screen for 2 minutes is boring as shit

File: code/modules/mob/living/carbon/alien/special/facehugger.dm [hunk 1]
```diff
@@ -182,8 +182,9 @@
 
 	if(!sterile)
 		M.take_bodypart_damage(strength,0) //done here so that humans in helmets take damage
-		M.Unconscious(MAX_IMPREGNATION_TIME/0.3) //something like 25 ticks = 20 seconds with the default settings
-
+		M.emote("scream")
+		M.Paralyze(MAX_IMPREGNATION_TIME/0.3) //something like 25 ticks = 20 seconds with the default settings
```
Comment (Ghommie) [inline code/modules/mob/living/carbon/alien/special/facehugger.dm:186]: Please update the comment.

**Review by Ghommie — CHANGES_REQUESTED** [review]

**Pull request closed without merging**